cmake/presets/macos-multiarch.cmake

```diff
@@ -10,5 +10,3 @@ set(CMAKE_CXX_FLAGS_RELEASE "-O3 -DNDEBUG" CACHE STRING "" FORCE)
 set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG" CACHE STRING "" FORCE)
 
 set(BUILD_MPI FALSE CACHE BOOL "" FORCE)
-set(BUILD_SHARED_LIBS FALSE CACHE BOOL "" FORCE)
```

Comment:
Just curious, what is the reason behind the changes here?

Reply:
Shared libraries work just as well as static libraries when compiling for macOS with apple and intel CPUs at the same time. So there is no need to forcibly turn this off. This preset was originally used for the LAMMPS .dmg files where the binaries were supposed to be linked to a static LAMMPS library.